app/gui/view/src/project.rs

```diff
@@ -109,9 +129,17 @@ ensogl::define_endpoints! {
         execution_context_restart(),
         toggle_read_only(),
         set_read_only(bool),
+        /// Started creation of a new node using the AI searcher.
+        start_node_creation_with_ai_searcher(),
+        /// Started creation of a new node using the Component Browser.
+        start_node_creation_with_component_browser(),
+        /// Accept the current input of the searcher.
+        accept_searcher_input(),
```

Comment:
We still have this input, just not assigned to any shortcut.

And my original comment was mostly about name, as the name did not say about accepting _AI_ searcher only. But if the shortcut from the original searcher works, we can go with it (just remove this input then).